.github/workflows/build-pr-status-check.yml

```
@@ -31,7 +31,7 @@ jobs:
       run: |
         $pullRequestNumber = "${{ github.ref }}".Replace("refs/pull/", "").Replace("/merge", "");
         Write-Host "Pull Request Number: $pullRequestNumber";
-        Write-Host "::set-output name=pr-number::$pullRequestNumber";
+        "pr-number=$pullRequestNumber" >> $GITHUB_OUTPUT;
```

Need to fix the environment variable **GITHUB_OUTPUT**.  Fix this for ALL workflows

```suggestion
        "pr-number=$pullRequestNumber" >> $env:GITHUB_OUTPUT;
```